conditional/templates/intro_eval_slideshow.html

```diff
@@ -47,8 +47,9 @@ <h3>{{m['technical_seminars']|length}}</h3>
               </div>
             </div>
           </div>
-        <h4><span class="icon glyphicon glyphicon-{% if m['freshman_project'] == 'Passed' %}ok passed{%else%}remove{% endif %}" aria-hidden="true"></span> Freshman Project</h4>
-        
+
+        <h4>{% if m['freshman_project'] != 'Pending' %}<span class="icon glyphicon glyphicon-{% if m['freshman_project'] == 'Passed' %}ok passed{%else%}remove{% endif %}" aria-hidden="true"></span> Freshman Project{% endif %}</h4>
```

Comment:
Can you make this three lines to show the layered logic? That would be much easier to read.